ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+Issue tracker is **ONLY** used for reporting bugs. Support or new features should be discussed on [Lava's Discord channel](https://discord.gg/5VcqgwMmkA).
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
```

Comment:
Consider moving this section below "steps to reproduce", and write that everything after "steps to reproduce" is optional. This lowers the effort required to submit bug reports. 
If you are blasted with low quality bug reports, you can then change the template to require more details.

Reply:
Done 👍